script.grab.fanart/changelog.txt

```diff
@@ -0,0 +1,163 @@
+Version 0.15.1
```

Comment:
This is no longer parsed by kodi, please drop it from the submission and use only the news tag in add-on.xml

Reply:
Will do. I like to keep it as part of the repo for my own history but can remove from submission.